.github/workflows/zowe-exlorer-ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Zowe Explorer CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [10.x, 12.x]
+        # order operating systems from best to worst
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - run: npm install
+
+    - name: Audit from Windows / Node 12.x
+      run: npm audit --audit-level=moderate
+      if: matrix.os == 'windows-latest' && matrix.node-version == '12.x'
```

Comment:
Do you think this if is necessary? I think we could do well without it. I do not think that it is a significant performance bottleneck.

Reply:
I dont think its a performance hit either.  I was worried about storage consumption and saving results from each combination of Node + OS.  I'll plan to remove this check though unless anyone objects.

Reply:
I was going to ask about the `if` there but saw this conversation.
fyi, we may run into problems due to npm-audit servers randomly failing.

Reply:
we can go whichever way the team prefers 😄 

this was an interesting exercise, I have a decent number of examples of trying things in different ways